CHANGELOG.md

```diff
@@ -1,5 +1,69 @@
 # Changelog
 
+# [v0.10.0](https://github.com/kubermatic/kubeone/releases/tag/v0.10.0) - 2019-09-26
```

Comment:
I'd put the whole changelog here, since v0.9.2, but in the release notes put the link to this files and changelog since v0.10.0-alpha.3. Does that make sense?